public/js/app/components/MemoMenu.jsx

```diff
@@ -167,32 +145,41 @@ function MemoMenu(props) {
       {/** *********************************************************************************** */}
       {/*                                  SEMESTER DROPDOWN                                  */}
       {/** *********************************************************************************** */}
-      <Dropdown isOpen={dropdownOpen} toggle={toggleDropdown} className="select-semester">
-        <h2 className="section-50">{translate.header_memo_menu}</h2>
 
+      <div className="inline-flex flex-column padding-top-30">
+        <h2 className="section-50">{translate.header_memo_menu}</h2>
         <FormHeaderAndInfo infoId="info_select_semester" header={selectSemester} translate={translate} />
+      </div>
+      <div className="col-4 nopadding">
+        <form>
+          <div className="form-group">
+            <div className="form-select">
+              <div className="select-wrapper">
+                <select
+                  className="form-control"
+                  id="semesterDropdownControl"
+                  aria-label={translate.select_semester}
```

Comment:
in this component on line 131 we created (destructed) const `selectSemester` from `translate.select_semester`. It means you can reuse `selectSemester` instead of `translate.select_semester` here in `select`'s `aria-label`, `defaultValue` and `option's value`